### PR TITLE
Updated Readme/contribution guidelines/license/templates

### PR DESCRIPTION
**Pull request by himanshu219**

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (duchatran) [inline .github/CODE_OF_CONDUCT.md:1]: Why these files are under .github and not at the top level?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (duchatran) [inline LICENSE:189]: Don't we need to put Sumo Logic Inc. here?

Comment (himanshu219) [reply LICENSE:189]: yes I copied it from python sdk seems that project also doesn't have this.

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (himanshu219) [reply .github/CODE_OF_CONDUCT.md:1]: it's a convention folder. Also the main idea of these files is github shows them when you create issue or PR in the side panel, I think people rarely read it directly from these files. And we also reference them in Readme but I can move to root for visibility. rest of TEMPLATE.md files will remain in .github folder right?

File: LICENSE (hunk 2 above)
Comment (himanshu219) [reply LICENSE]: fixed

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (himanshu219) [reply .github/CODE_OF_CONDUCT.md]: done

**Pull request merged**